scripts/update-cargo-version.js

```diff
@@ -0,0 +1,31 @@
+/// @ts-check
+
+const versionPattern = /[ \t]*version\s*=\s*"([^"]+)"/;
```

Comment:
Using a regex is faster and easier to understand than parsing and patching the entire abstract syntax tree of the Cargo.toml document.